lib/board.rb

```diff
@@ -32,8 +37,45 @@ def free_positions
     flat_board.each_index.select { |index| empty?(flat_board[index]) }
   end
 
+  def content
```

Comment:
Oh. Yeah it's parsed every time it's called so, every times the tests need it and it will be also use by the different interface (cli/web) instead of the current double array that they using